src/components/Panels/index.ts

```diff
@@ -0,0 +1,11 @@
+import Panel from './Panel.vue'
+import PanelContainer from './PanelContainer.vue'
+
+import { App } from 'vue'
+
+Panel.install = (app: App) => {
+  app.component('MPanel', Panel)
+  app.component('MPanelContainer', PanelContainer)
```

Comment:
Is there a reason to export this? It doesn't seem like this should ever be called without the plugin but I might be wrong

Reply:
Nope, was just following the other installers. Removed 👍